low-level/framework/platforms/server/platform.py

```diff
@@ -69,8 +74,8 @@ def get_server_details(self):
             "Board Part Number": "",
             "Product Name": "",
             "Product Part Number": "",
-            "Manufacturer": self.get_manufacturer_name(),
-            "OS": self.get_os()
+            "Manufacturer": self._ipmi.get_manufacturer_name(),
+            "OS": Utility().get_os()
             }
         cmd = "fru print"
         prefix = "FRU Device Description : Builtin FRU Device (ID 0)"
```

Comment:
I found an issue during integration.

Line 88-89
```
            if search_res:
                search_res = search_res.group()
```
Please replace this by
```
search_res = search_res.group() if search_res else ""
```

Reply:
Done